common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -48,6 +48,52 @@ class SequenceFields(object):
         scope=Scope.content,
     )
 
+    is_time_limited = Boolean(
+        display_name=_("Is Timed Limited"),
+        help=_(
+            "This field indicates if this courseware container will have a time limit to view/interact. "
+        ),
+        default=False,
+        scope=Scope.settings,
+    )
+
+    default_time_limit_mins = Integer(
+        display_name=_("Time Limit in Minutes"),
+        help=_(
+            "The number of minutes a user will have to view this courseware."
+            "once he/she has started."
```

Comment:
I think it's OK to do without this last phrase? Strike "once he/she has started."